open_instruct/vllm_utils3.py

```diff
@@ -452,17 +454,22 @@ def _process_request(self, request):
 
     def _add_initial_requests(self, prompts, sampling_params, n_samples, training_step):
         """Add initial requests to the engine."""
+        if os.environ.get("VLLM_USE_V1") == "0":
```

Comment:
You could check this in the args post_init and then assert that `enable_prefix_caching` is false

Reply:
Nice! Done.